modules/ignition/resources/services/kubelet.service

```diff
@@ -11,6 +11,9 @@ ExecStart=/usr/bin/hyperkube \
     --bootstrap-kubeconfig=/etc/kubernetes/kubeconfig \
     --kubeconfig=/var/lib/kubelet/kubeconfig \
     --rotate-certificates \
+    --container-runtime=remote \
+    --container-runtime-endpoint=unix:///var/run/crio/crio.sock \
```

Comment:
This would also need to align with a change in the pod-checkpointer daemonset: https://github.com/kubernetes-incubator/bootkube/blob/master/cmd/checkpoint/main.go#L21

And if tests pass without this change - I'd be worried (and we should make sure checkpoint tests are actually running.

Reply:
Thanks. I added the command line arg [here](https://github.com/coreos-inc/tectonic-operators/pull/457/files#diff-a00818a744cc627c8714b2e70de8f279R30).